account_invoice_report_due_list/tests/test_invoice_report_due_list.py

```diff
@@ -108,7 +108,20 @@ def test_due_list(self, move_type="out_invoice"):
         self.assertRegex(str(res[0]), date_due_format)
         self.assertRegex(str(res[0]), "75.0")
 
+    @mute_logger("odoo.models.unlink")
     def test_due_list_currency_extra(self, move_type="out_invoice"):
+        # We leave the rate of the extra currency at 1 so that the amounts to be
+        # checked are as expected.
+        currency_extra_rates = self.env["res.currency.rate"].search(
+            [("currency_id", "=", self.currency_extra.id)]
+        )
+        currency_extra_rates.unlink()
```

Comment:
No need of extra variable and less lines of code:

```suggestion
        self.env["res.currency.rate"].search(
            [("currency_id", "=", self.currency_extra.id)]
        ).unlink()
```